Directory.Build.props

```diff
@@ -20,26 +20,26 @@
   <!-- ******************************************************************************* -->
   <!-- Newtonsoft -->
   <!-- ******************************************************************************* -->
-  <!-- NOTE: we are referencing a version that is installed with VS2017 RTM. Newer versions  of VS ship with newer versions
-         of the Newtonsoft assembly and have binding redirects. This means that we don't need to ship the Newtonsoft dll
-         ourselves. 
-         However, the SonarAnalyzers from v4.25+ also depend on a different version of the Newtonsoft assembly which is
-         newer version than the one that ships in VS. This means the VSIX needs to contain that version of the Newtonsoft
-         assembly so that some of the analyzer rules can work (Integration.Vsix.csproj handles this).
+  <!-- NOTE: We're specifying the version we want to use and are embedding it in the VSIX.
+         However, VS ships with multiple verions of NewtonSoft.Json.dll and has binding redirects
+         in devenv.exe.config that can change the version is loaded at runtime. However, this
+         should not cause a problem as we will end up with a compatible version.
+
+         Also, VS2019 ships with NewtonSoft v12.x i.e. we are using a newer version. This
+         should not cause a problem since VS can load both versions.
+         
+         Finally, NewtonSoft.Json v12 has vulnerabilities reported against it which Mend
+         will complain about. By referencing v13.x we avoid those warnings.
          
-         See #2474 for more info.
+         See #3098 for more info.
```

Comment:
Updated comment: we're now embedding a specific version of the NewtonSoft dll, and we're using the same version for both VS2019 and VS2022.

Reply:
VS2019 ships with NewtonSoft.Json.dll v12.x. We're installing v13 on the same machine. When VS runs it will load both versions:

![image](https://github.com/SonarSource/sonarlint-visualstudio/assets/10757559/c767e0c7-a89a-47c4-aace-7286b6bb01fa)

Reply:
With VS2022, it will only load one version:
![image](https://github.com/SonarSource/sonarlint-visualstudio/assets/10757559/4ae4ec16-bc59-425e-a9f5-16f56709b72a)